### DISCUSSION TITLE
Use tailwind

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#14170] [Feature Requests]
When we're writing extensions, it's almost impossible to reuse css classes.

The use of tailwind in the project would be very interesting to improve the DX of writing extensions.

**Comment by rijkvanzanten** [comment]
> When we're writing extensions, it's almost impossible to reuse css classes.

That's because there's currently no CSS classes meant to be reused like that 🙂 

> The use of tailwind in the project would be very interesting to improve the DX of writing extensions.

Agree to disagree. I'd much prefer to come up with a system of well scoped variables to reuse in CSS, that way extension creators can write the styles in any way they prefer, while still reusing the globally influenced styles 👍🏻

**Reply by rijkvanzanten** [reply]
>  I'd much prefer to come up with a system of well scoped variables to reuse in CSS

(which is what https://github.com/directus/directus/pull/13247 aims to finalize)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
